### PR TITLE
TBD-6087: fixed

### PR DESCRIPTION
**Pull request by mtuhai**
https://jira.talendforge.org/browse/TBD-6087

### CODE REVIEW

File: main/plugins/org.talend.designer.components.localprovider/components/tHiveConnection/tHiveConnection_java.xml [hunk 1]
```diff
@@ -2628,6 +2628,50 @@
 				REQUIRED_IF="(HIVE_VERSION=='Cloudera_CDH5' AND STORE_BY_HBASE=='true') AND (DISTRIBUTION!='CUSTOM')" />
 			<!-- ############## -->
 
+			<!-- MAPR600 -->
+			<IMPORT NAME="hive-service-1.2.0.jar" MODULE="hive-service-1.2.0.jar"
```
Comment (bchen-talend) [inline main/plugins/org.talend.designer.components.localprovider/components/tHiveConnection/tHiveConnection_java.xml]: why do not use mapr600 plugin.xml to configure these dependencies?

File: main/plugins/org.talend.designer.components.localprovider/components/tHiveConnection/tHiveConnection_java.xml [hunk 2]
```diff
@@ -2679,6 +2679,10 @@
 			<IMPORT MODULE_GROUP="HIVE-LIB-MAPR50X_LASTEST"
 				REQUIRED_IF="HIVE_VERSION=='MAPR500' AND (DISTRIBUTION!='CUSTOM')" />
 
+			<!-- MAPR600 -->
+			<IMPORT MODULE_GROUP="HIVE-LIB-MAPR600"
```
Comment (lbourgeois) [inline main/plugins/org.talend.designer.components.localprovider/components/tHiveConnection/tHiveConnection_java.xml]: As @bchen-talend mentioned we should use mapr6 plugin.xml for this dependency

Comment (mtuhai) [reply main/plugins/org.talend.designer.components.localprovider/components/tHiveConnection/tHiveConnection_java.xml]: But I have already used mapr6 hive library from mapr6 plugin

Comment (mtuhai) [reply main/plugins/org.talend.designer.components.localprovider/components/tHiveConnection/tHiveConnection_java.xml]: This HIVE-LIB-MAPR600 library is from mapr6 plugin from tbd-se

Comment (bchen-talend) [reply main/plugins/org.talend.designer.components.localprovider/components/tHiveConnection/tHiveConnection_java.xml]: Hi @mtuhai 
As you define the HIVE-LIB-MAPR600 in mapr 600 distribution class, you should won't need to add this in here, no?

**Pull request closed without merging**